irene_plugin_telegram_face/plugin_telegram_audio_input.py

```diff
@@ -25,7 +25,21 @@ class TelegramAudioInputPlugin(MagicPlugin):
     """
 
     name = 'telegram_input_audio'
-    version = '0.1.0'
+    version = '0.1.1'
+
+    config_comment = """
+    Настройки приёма голосовых сообщений из Telegram.
+
+    Доступны следующие параметры:
+    - `recogniseTextReply`  - слать сообщения с распознаным текстом из голосового сообщения
```

Comment:
```suggestion
    - `recognizeTextReply`  - слать сообщения с распознанным текстом из голосового сообщения
```

Опечатки в названии параметра и в описании